app/views/shared/_header.html.erb

```diff
@@ -1,5 +1,5 @@
 <nav class="navbar navbar-expand-sm">
-  <a class="navbar-brand" href="#"><%= link_to image_tag('sul-logo.png', height: '30'), :root %></a>
+  <a class="navbar-brand" href="#"><%= link_to image_tag('sul-logo.png', height: '30', alt: 'Stanford Libraries home'), :root %></a>
```

Comment:
But this is a link to pre-assembly root, not Stanford Libraries home.

Reply:
Argh, I can't believe I missed that. Why are we using a Libraries logo in this way? @aaron-collier @edsu

Reply:
Only reason I caught this is because it was similarly broken H2.

Reply:
It seems normal to link an app icon to the app's home? But we don't have an app icon for Preassembly. It does seem like in the absence of a app icon we should link an icon for SUL to SUL's homepage? @astridu do you have an opinion?

Reply:
Or maybe we could have a basic text logo that says Preassembly? Since "Preassembly" doesn't appear anywhere but the URL currently and title in the browser, and might be nice to have on the page. We have "Accessioning" but that's not what users call the app.